Started by user [8mha:////4C8zTO/v0UlG+vQ5Fjm5jTf9kR93yL5yxTMP2X04T9v8AAAAlh+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAyWEgYu/dLi1CL9ovz8EgDKVWzSvwAAAA==[0mroot
Running in Durability level: MAX_SURVIVABILITY
[8mha:////4DOlrZhma2TKYqx8mtRBwGXVE8Xt4z3OJyo/zi+ICDdzAAAAoh+LCAAAAAAAAP9tjTEOwjAQBM8BClpKHuFItIiK1krDC0x8GCfWnbEdkooX8TX+gCESFVvtrLSa5wtWKcKBo5UdUu8otU4GP9jS5Mixv3geZcdn2TIl9igbHBs2eJyx4YwwR1SwULBGaj0nRzbDRnX6rmuvydanHMu2V1A5c4MHCFXMWcf8hSnC9jqYxPTz/BXAFEIGsfuclm8zQVqFvQAAAA==[0m[Pipeline] Start of Pipeline
[8mha:////4B4K7nOhvaaLmbeE3w0NEhE1GAALwLDjakw9plXgkY6XAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycohUghExsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jduZBmjwAAAAA==[0m[Pipeline] node
Still waiting to schedule task
Waiting for next available executor on â€˜[8mha:////4EheNQ7D4c27QswDGPiDpGtf9rfQF1HauEmyd+/REsZ/AAAAoh+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAyeEgZx/eT83ILSktQiICOvpCg/Jye1SDdLHwCRPP8lzAAAAA==[0mcontroller-jâ€™
Running on [8mha:////4EheNQ7D4c27QswDGPiDpGtf9rfQF1HauEmyd+/REsZ/AAAAoh+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAyeEgZx/eT83ILSktQiICOvpCg/Jye1SDdLHwCRPP8lzAAAAA==[0mcontroller-j in /root/.jenkins/workspace/test-test-iperf-SRIOV-ipv4-stress-2hours
[8mha:////4NUxBMzN6CtFAbQ4pdo7pH8ZuwVHigLf0SdP3FcjQTwyAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gA0xsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jfoP95RwAAAAA==[0m[Pipeline] {
[8mha:////4OCB1+Up79PmV3LnMeomKr9r+JFCKkFjAfTpr2UorbLOAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gQkxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jc09154wAAAAA==[0m[Pipeline] stage
[8mha:////4N6KQN35K8CT/Pcr9a+R4oOEu4uEh3phvpB1t67zd2hmAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0ggUxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jek7ggRwAAAAA==[0m[Pipeline] { (clone)
[8mha:////4KpTcsSlLG7yEvo1vGv2QG1z86HMYkLASFXVuKwhd7SZAAAAoh+LCAAAAAAAAP9tjTEOAiEURD9rLGwtPQTbaWGsbAmNJ0AWEZb8zwLrbuWJvJp3kLiJlZNMMm+a93rDOic4UbLcG+wdZu14DKOti0+U+lugiXu6ck2YKRguzSSpM+cFJRUDS1gDKwEbgzpQdmgLbIVXD9UGhba9lFS/o4DGdQM8gYlqLiqVL8wJdvexy4Q/z18BzLEA29ce4gfg7KmOvAAAAA==[0m[Pipeline] sh
+ git clone -b develop https://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@github.com/Jiz1028/applications.validation.network-platform.npxtester.git npxtester
Cloning into 'npxtester'...
[8mha:////4ChDQWb8Lvr0RoXQ1L8EY4p37+Po0Q8Mb8bchs8JeqDJAAAAoh+LCAAAAAAAAP9tjTEOAiEURD9rLGwtPQTbGRNjZUtoPAGyiLDkfxZYdytP5NW8g8RNrJxkknnTvNcb1jnBiZLl3mDvMGvHYxhtXXyi1N8CTdzTlWvCTMFwaSZJnTkvKKkYWMIaWAnYGNSBskNbYCu8eqg2KLTtpaT6HQU0rhvgCUxUc1GpfGFOsLuPXSb8ef4KYI6xADvU7j9Dg2gqvAAAAA==[0m[Pipeline] }
[8mha:////4J0p4zt/+On27QitkvU5TFB8M2wwWYWaAHIuYiUer4fkAAAAoh+LCAAAAAAAAP9tjTEOAiEURD9rLGwtPQRbWRhjZUtoPAGyiLDkfxZYdytP5NW8g8RNrJxkknnTvNcb1jnBiZLl3mDvMGvHYxhtXXyi1N8CTdzTlWvCTMFwaSZJnTkvKKkYWMIaWAnYGNSBskNbYCu8eqg2KLTtpaT6HQU0rhvgCUxUc1GpfGFOsLuPXSb8ef4KYI6xADvU7j9J+wGOvAAAAA==[0m[Pipeline] // stage
[8mha:////4Ea3/4ZYuTM98yMZ9UntzMXOyQ2ItSOm6XAUUO4QqkITAAAAph+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQDEh2iorXc8AITG+PEugv2haTiRXyNPxCIRMVWOyut5vmCMic4UPKycdgGzHWQXez91ORAqb1EGmRDZ1kTZopOajdosu44oyZ2MEcUsFCwdFhHygE9w0o15m6qaNBXJ07TtldQBHuDBwg1mdkk/sKYYH3tbSb8ef4KYOwYxI6h2G4+x/INtuQqUcEAAAA=[0m[Pipeline] stage
[8mha:////4IJd0qxfNFkcgwUT9OFke9rxaJjwCH2G9mHJ+5FMmbLfAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+DOrAycoiUzoiJNerCCUITQtLo/zZJSSdOxNW4Ay2VmPBg2Zas93pDGQOcKBjuNHYWY2t570czJ54pdDdPmTu68pYwkte80bkhpc9rbShpWFUw2AjYamw9RYsmwU44+ZCVl2iqSwrzdhTArBrgCYWYyUmG9C1TgP19VJHwx/kLgKlPwOrDYvXyLD8BobDcwgAAAA==[0m[Pipeline] { (set args)
[8mha:////4KACpsxoNqug+yeGIE2K55MD+IyPDLfbYOrean7h1Zl/AAAAox+LCAAAAAAAAP9tjTESwiAURH/iWNhaeggysXWsbBkaT4AJIoT5nwCRVJ7Iq3kH0cxYucXOvm3e8wXrGOBIQTOrcDAYO8O8m3RZLFMYro4ys3RhHWEkp5hQWVCvTgsKSgqWVDWsOGwUdo6iQZ1gy628y8ZJ1M05hfIdONSmH+EBFS/mJEP6whxgd5v6SPjz/BXA7BPUbfupvX8DpQJrNb4AAAA=[0m[Pipeline] sh
+ python /root/.jenkins/workspace/test-test-iperf-SRIOV-ipv4-stress-2hours/npxtester/conf/change_args.py '{"dpdk_dir":"/opt/APP/utility/DPDK_install/dpdk-stable-20.11.1/","vm_image_conf":{"image1":"uvm1.img","image2":"uvm2.img","image3":"rhel84rt.img","image4":"rhel84rt2.img","image_dir":"/home/image"},"sut_conf":{"host":"10.112.97.62","pf1":"ens2f0","os":"rehat"},"sut2_conf":{"host":"10.112.97.60","pf1":"ens2f1"},"run_type":"ipv4"}'
[8mha:////4KMKnj+jwomtBZRlOM1zDNsT7NVP2UylJUBAmwQdpqOFAAAAox+LCAAAAAAAAP9tjTESwiAURH/iWNhaeggyGUvHypah8QSYIEKY/wkQSeWJvJp3EM2MlVvs7NvmPV+wjgGOFDSzCgeDsTPMu0mXxTKF4eooM0sX1hFGcooJlQX16rSgoKRgSVXDisNGYecoGtQJttzKu2ycRN2cUyjfgUNt+hEeUPFiTjKkL8wBdrepj4Q/z18BzN4nqNv9p9o3geV5574AAAA=[0m[Pipeline] }
[8mha:////4PHbOgS4fTQzlO31aLQygq6Gmyy7LbpReNAbbgHuNOsPAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOICEh2iSmu54QUmNsaJdWdsh6TiRXyNPxCIRMUWq51t5vmCZYpw5GixNdQ5So3D4Hs7LRw4dhfPA7Z8xoYpsTcozSBZm3pGydnAnKKEhYCVocZzcmQzrEWr7qryimx1ynH6DgJKp2/wgEJM5qxi/sIYYXPtdWL6ef4KYAwhQ7nbf2r7Bq/le42+AAAA[0m[Pipeline] // stage
[8mha:////4MZyk33+V/AiT4fn8n6G3pw2AKptiha+GL2FVGy8vSODAAAAph+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjJKgQFa3lhheY2Bgn1l2wLyQVL+Jr/IFAJCq22llpNc8XlDnBgZKXjcM2YK6D7GLvpyYHSu0l0iAbOsuaMFN0UrtBk3XHGTWxgzmigIWCpcM6Ug7oGVaqMXdTRYO+OnGatr2CItgbPECoycwm8RfGBOtrbzPhz/NXAGPHILYMxWb3OZZvBXXnAcEAAAA=[0m[Pipeline] stage
[8mha:////4IbAYeEfzIYrbjXur3hhLHFS6T3zmMsYQQAYadAUrpZ/AAAApx+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohUDLAgpq5RFk4QmhDSRv+Xn5R24kRcjTvQUokJD5ZtyXqvNxSR4UTsRGOx9RhrL7rQuymJgbi9BhpEQxdRE0YKVig7KDK2WqqiZGFRlsNKwtpiHSh6dAk2stEPXQaNrjwnnrajhNybOzwhkxM5aU7fMjJsb72JhD/OXwCMXYJ8t5/tMD+LD/s94rDCAAAA[0m[Pipeline] { (run nic case)
[8mha:////4FPk2mTBzwof/oyzK3/Y2W4GOSwpNdCdlxh/EpNq5ACcAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiGigQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGPIUG53n9qHN27Iu42+AAAA[0m[Pipeline] sh
+ pytest /root/.jenkins/workspace/test-test-iperf-SRIOV-ipv4-stress-2hours/npxtester/Tests/testNics -s -m SriovVfIperf --html=/root/testconfig/testlogs/HtmlReport/sriov_iperfvf_2hours_ipv4.html --self-contained-html
============================= test session starts ==============================
platform linux -- Python 3.6.8, pytest-6.2.4, py-1.11.0, pluggy-0.13.1
rootdir: /root/.jenkins/workspace/test-test-iperf-SRIOV-ipv4-stress-2hours/npxtester/Tests/testNics, configfile: pytest.ini
plugins: metadata-1.11.0, html-3.1.1
collected 16 items / 15 deselected / 1 selected

npxtester/Tests/testNics/test_network_sriov_stress.py 2022-02-11 09:47:37 - NicTestFixture.py - [line:28] - root - INFO - Starting to run test_PI_Virtualization_SRIOV_VF_Stress_L[uvm2.img-uvm1.img]
2022-02-11 09:47:37 - NicTestFixture.py - [line:29] - root - INFO - setup log for testing in the fixture class fbefore
2022-02-11 09:47:37 - test_network_sriov_stress.py - [line:31] - root - INFO - SSH SUT and setup configurations in the setup function.
2022-02-11 09:47:37 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-02-11 09:47:37 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-02-11 09:47:37 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.112.97.62
2022-02-11 09:47:37 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-02-11 09:47:37 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-02-11 09:47:37 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-02-11 09:47:37 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.112.97.60
2022-02-11 09:47:40 - sshInstance.py - [line:44] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-stable-20.11.1/usertools/dpdk-devbind.py -s | grep ens2f0 | awk '{print $1;}'
2022-02-11 09:47:41 - sshInstance.py - [line:48] - root - INFO - 0000:4b:00.0

2022-02-11 09:47:46 - test_network_sriov_stress.py - [line:87] - root - INFO - pf_id was found: 0000:4b:00.0

2022-02-11 09:47:46 - test_network_sriov_stress.py - [line:787] - root - INFO - format_pf_id is 0000\:4b\:00.0

2022-02-11 09:47:46 - test_network_sriov_stress.py - [line:790] - root - INFO - pf_id_key: 4b
2022-02-11 09:47:52 - sshInstance.py - [line:92] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket
This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register
Last login: Fri Feb 11 09:54:39 2022 from 10.238.9.206
[root@WW25ICX2 ~]# ifconfig ens2f0 192.168.10.10
[root@WW25ICX2 ~]# 
2022-02-11 09:47:55 - test_network_sriov_stress.py - [line:283] - root - INFO - Successfully assigned ip to ethernet: ens2f0
2022-02-11 09:48:01 - sshInstance.py - [line:92] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket
This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register
Last login: Fri Feb 11 08:58:20 2022 from 10.112.97.60
[root@localhost ~]# ifconfig ens2f1 192.168.10.11
[root@localhost ~]# 
2022-02-11 09:48:04 - test_network_sriov_stress.py - [line:283] - root - INFO - Successfully assigned ip to ethernet: ens2f1
2022-02-11 09:48:07 - sshInstance.py - [line:44] - root - INFO - echo 4 > /sys/class/net/ens2f0/device/sriov_numvfs
2022-02-11 09:48:07 - sshInstance.py - [line:48] - root - INFO - 
2022-02-11 09:48:12 - test_network_sriov_stress.py - [line:101] - root - INFO - Successfully Virtualized PF to VFs.
2022-02-11 09:48:15 - sshInstance.py - [line:44] - root - INFO -  lspci | grep "Virtual" | awk '{print $1;}'list
2022-02-11 09:48:15 - sshInstance.py - [line:48] - root - INFO - 4b:01.0
4b:01.1
4b:01.2
4b:01.3

2022-02-11 09:48:20 - test_network_sriov_stress.py - [line:115] - root - INFO - Succssfully Gotten vf_id list:

2022-02-11 09:48:23 - sshInstance.py - [line:44] - root - INFO -  modinfo vfio-pci | grep "vfio_pci" | awk '{ print $2;}'
2022-02-11 09:48:23 - sshInstance.py - [line:48] - root - INFO - vfio_pci

2022-02-11 09:48:28 - test_network_sriov_stress.py - [line:168] - root - INFO - Successfuly modeprobe vfio-pci
2022-02-11 09:48:31 - sshInstance.py - [line:44] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-stable-20.11.1/usertools/dpdk-devbind.py -b vfio-pci 4b:01.0
2022-02-11 09:48:32 - sshInstance.py - [line:48] - root - INFO - 
2022-02-11 09:48:37 - test_network_sriov_stress.py - [line:147] - root - INFO - Successfuly bound VF to vfio-pci
2022-02-11 09:48:40 - sshInstance.py - [line:44] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-stable-20.11.1/usertools/dpdk-devbind.py -b vfio-pci 4b:01.1
2022-02-11 09:48:40 - sshInstance.py - [line:48] - root - INFO - 
2022-02-11 09:48:45 - test_network_sriov_stress.py - [line:147] - root - INFO - Successfuly bound VF to vfio-pci
2022-02-11 09:48:45 - test_network_sriov_stress.py - [line:814] - root - INFO - lspci -Dnn | grep 4b:01.1
2022-02-11 09:48:48 - sshInstance.py - [line:44] - root - INFO - lspci -Dnn | grep 4b:01.1
2022-02-11 09:48:48 - sshInstance.py - [line:48] - root - INFO - 0000:4b:01.1 Ethernet controller [0200]: Intel Corporation Ethernet Adaptive Virtual Function [8086:1889] (rev 02)

2022-02-11 09:48:53 - test_network_sriov_stress.py - [line:825] - root - INFO - Successfully get vf device id: 8086:1889
2022-02-11 09:48:56 - sshInstance.py - [line:44] - root - INFO - virsh net-info default | grep "Active" | awk '{print $2;}'
2022-02-11 09:48:56 - sshInstance.py - [line:48] - root - INFO - yes

2022-02-11 09:49:01 - test_network_sriov_stress.py - [line:177] - root - INFO - Default network is active? yes

2022-02-11 09:49:05 - sshInstance.py - [line:69] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vcpus=5 ---network network=default --hostdev pci_0000_4b_01_0 --boot hd --disk path=/home/iimage/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5900 --noaautoconsole

** (process:72424): [1;33mWARNING[0m **: [34m09:57:57.279[0m: Entity http://pcisig.com/pci/1B36/0100 referenced but not defined
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:4b:01.0 is in use by driver QEMU, domain VMtest1
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest1
otherwise, please restart your installation.
[root@WW25ICX2 ~]# 
2022-02-11 09:49:08 - test_network_sriov_stress.py - [line:209] - root - ERROR - Failed to create a VM: virt-install --name VMtest1 --ram 4096 --cpu host --vcpus=5 ---network network=default --hostdev pci_0000_4b_01_0 --boot hd --disk path=/home/iimage/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5900 --noaautoconsole

** (process:72424): [1;33mWARNING[0m **: [34m09:57:57.279[0m: Entity http://pcisig.com/pci/1B36/0100 referenced but not defined
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:4b:01.0 is in use by driver QEMU, domain VMtest1
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest1
otherwise, please restart your installation.
[root@WW25ICX2 ~]# 
2022-02-11 09:49:08 - test_network_sriov_stress.py - [line:210] - root - ERROR - Failed to create a VM: 
2022-02-11 09:49:12 - sshInstance.py - [line:69] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vcpus=5 ---network network=default --hostdev pci_0000_4b_01_0 --boot hd --disk path=/home/iimage/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5900 --noaautoconsole

** (process:72628): [1;33mWARNING[0m **: [34m09:58:04.289[0m: Entity http://pcisig.com/pci/1B36/0100 referenced but not defined
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-02-11 09:49:14 - sshInstance.py - [line:69] - root - INFO - Domain creation completed.
[root@WW25ICX2 ~]# 
2022-02-11 09:49:17 - test_network_sriov_stress.py - [line:212] - root - INFO - Successfully created a VM: VMtest1
2022-02-11 09:49:21 - sshInstance.py - [line:69] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vcpus=5 ---network network=default --hostdev pci_0000_4b_01_1 --boot hd --disk path=/home/iimage/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5901 --noaautoconsole

** (process:72671): [1;33mWARNING[0m **: [34m09:58:13.299[0m: Entity http://pcisig.com/pci/1B36/0100 referenced but not defined
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:4b:01.1 is in use by driver QEMU, domain VMtest2
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest2
otherwise, please restart your installation.
[root@WW25ICX2 ~]# 
2022-02-11 09:49:24 - test_network_sriov_stress.py - [line:209] - root - ERROR - Failed to create a VM: virt-install --name VMtest2 --ram 4096 --cpu host --vcpus=5 ---network network=default --hostdev pci_0000_4b_01_1 --boot hd --disk path=/home/iimage/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5901 --noaautoconsole

** (process:72671): [1;33mWARNING[0m **: [34m09:58:13.299[0m: Entity http://pcisig.com/pci/1B36/0100 referenced but not defined
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:4b:01.1 is in use by driver QEMU, domain VMtest2
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest2
otherwise, please restart your installation.
[root@WW25ICX2 ~]# 
2022-02-11 09:49:24 - test_network_sriov_stress.py - [line:210] - root - ERROR - Failed to create a VM: 
2022-02-11 09:49:28 - sshInstance.py - [line:69] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vcpus=5 ---network network=default --hostdev pci_0000_4b_01_1 --boot hd --disk path=/home/iimage/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5901 --noaautoconsole

** (process:72675): [1;33mWARNING[0m **: [34m09:58:20.309[0m: Entity http://pcisig.com/pci/1B36/0100 referenced but not defined
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-02-11 09:49:30 - sshInstance.py - [line:69] - root - INFO - Domain creation completed.
[root@WW25ICX2 ~]# 
2022-02-11 09:49:33 - test_network_sriov_stress.py - [line:212] - root - INFO - Successfully created a VM: VMtest2
2022-02-11 09:49:36 - sshInstance.py - [line:44] - root - INFO - virsh domifaddr VMtest1 | grep '192.168' | awk '{print $4;}'
2022-02-11 09:49:36 - sshInstance.py - [line:48] - root - INFO - 192.168.122.233/24

2022-02-11 09:49:41 - test_network_sriov_stress.py - [line:224] - root - INFO - Successfully got VMtest1 IP: 192.168.122.233/24

2022-02-11 09:49:44 - sshInstance.py - [line:44] - root - INFO - virsh domifaddr VMtest2 | grep '192.168' | awk '{print $4;}'
2022-02-11 09:49:44 - sshInstance.py - [line:48] - root - INFO - 
2022-02-11 09:49:49 - test_network_sriov_stress.py - [line:229] - root - ERROR - not found IP of VMtest2
2022-02-11 09:50:53 - sshInstance.py - [line:44] - root - INFO - virsh domifaddr VMtest2 | grep '192.168' | awk '{print $4;}'
2022-02-11 09:50:53 - sshInstance.py - [line:48] - root - INFO - 192.168.122.130/24

2022-02-11 09:50:58 - test_network_sriov_stress.py - [line:224] - root - INFO - Successfully got VMtest2 IP: 192.168.122.130/24

2022-02-11 09:51:02 - sshInstance.py - [line:69] - root - INFO - ping -c 5 192.168.122.233
PING 192.168.122.233 (192.168.122.233) 56(84) bytes of data.
64 bytes from 192.168.122.233: icmp_seq=1 ttl=64 time=0.274 ms
2022-02-11 09:51:03 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 192.168.122.233: icmp_seq=2 ttl=64 time=0.364 ms
2022-02-11 09:51:04 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 192.168.122.233: icmp_seq=3 ttl=64 time=0.475 ms
2022-02-11 09:51:05 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 192.168.122.233: icmp_seq=4 ttl=64 time=0.251 ms
2022-02-11 09:51:06 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 192.168.122.233: icmp_seq=5 ttl=64 time=0.326 ms

--- 192.168.122.233 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4124ms
rtt min/avg/max/mdev = 0.251/0.338/0.475/0.079 ms
[root@WW25ICX2 ~]# 
2022-02-11 09:51:15 - sshInstance.py - [line:92] - root - INFO - ssh root@192.168.122.233
root@192.168.122.233's password: 
2022-02-11 09:51:24 - sshInstance.py - [line:92] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Thu 10 Feb 2022 09:00:12 PM EST
  System load:  0.0                Processes:             137
  Usage of /:   39.1% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.233
  Swap usage:   0%

372 updates can be installed immediately.
227 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Thu Feb 10 20:51:19 2022 from 192.168.122.1
root@localhost:~# 
2022-02-11 09:51:33 - sshInstance.py - [line:92] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}' [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            '
0000:00:05.0
root@localhost:~# 
2022-02-11 09:51:39 - test_network_sriov_stress.py - [line:322] - root - INFO - successfully got vf pci id of vm1: 0000:00:05.0
2022-02-11 09:51:45 - sshInstance.py - [line:92] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
root@localhost:~# 
2022-02-11 09:51:50 - test_network_sriov_stress.py - [line:336] - root - INFO - Successfully got vf device name of vm1: ens5
2022-02-11 09:51:56 - sshInstance.py - [line:92] - root - INFO - ifconfig ens5 192.168.10.12
root@localhost:~# 
2022-02-11 09:51:59 - test_network_sriov_stress.py - [line:283] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-02-11 09:52:05 - sshInstance.py - [line:92] - root - INFO - ifconfig ens5 down
root@localhost:~# 
2022-02-11 09:52:15 - sshInstance.py - [line:92] - root - INFO - ifconfig ens5 hw ether 1234567890ac
root@localhost:~# 
2022-02-11 09:52:24 - sshInstance.py - [line:92] - root - INFO - ifconfig ens5 up
root@localhost:~# 
2022-02-11 09:52:33 - sshInstance.py - [line:92] - root - INFO - nmcli device connect ens5
Command 'nmcli' not found, but can be installed with:
apt install network-manager
root@localhost:~# 
2022-02-11 09:52:42 - sshInstance.py - [line:92] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.12  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::1034:56ff:fe78:90ac  prefixlen 64  scopeid 0x20<link>
        ether 12:34:56:78:90:ac  txqueuelen 1000  (Ethernet)
        RX packets 1  bytes 342 (342.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 9  bytes 766 (766.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
root@localhost:~# 
2022-02-11 09:52:51 - sshInstance.py - [line:92] - root - INFO - exit
logout
Connection to 192.168.122.233 closed.
[root@WW25ICX2 ~]# 
2022-02-11 09:52:58 - sshInstance.py - [line:69] - root - INFO - ping -c 5 192.168.122.130
PING 192.168.122.130 (192.168.122.130) 56(84) bytes of data.
64 bytes from 192.168.122.130: icmp_seq=1 ttl=64 time=0.284 ms
2022-02-11 09:52:59 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 192.168.122.130: icmp_seq=2 ttl=64 time=0.463 ms
2022-02-11 09:53:00 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 192.168.122.130: icmp_seq=3 ttl=64 time=0.271 ms
2022-02-11 09:53:01 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 192.168.122.130: icmp_seq=4 ttl=64 time=0.529 ms
2022-02-11 09:53:02 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 192.168.122.130: icmp_seq=5 ttl=64 time=0.497 ms

--- 192.168.122.130 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4091ms
rtt min/avg/max/mdev = 0.271/0.408/0.529/0.112 ms
[root@WW25ICX2 ~]# 
2022-02-11 09:53:11 - sshInstance.py - [line:92] - root - INFO - ssh root@192.168.122.130
root@192.168.122.130's password: 
2022-02-11 09:53:20 - sshInstance.py - [line:92] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Thu 10 Feb 2022 09:02:08 PM EST
  System load:  0.0                Processes:             136
  Usage of /:   38.9% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.130
  Swap usage:   0%

372 updates can be installed immediately.
227 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Thu Feb 10 20:53:37 2022 from 192.168.122.1
root@localhost:~# 
2022-02-11 09:53:29 - sshInstance.py - [line:92] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}' [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            '
0000:00:05.0
root@localhost:~# 
2022-02-11 09:53:35 - test_network_sriov_stress.py - [line:322] - root - INFO - successfully got vf pci id of vm1: 0000:00:05.0
2022-02-11 09:53:41 - sshInstance.py - [line:92] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
root@localhost:~# 
2022-02-11 09:53:46 - test_network_sriov_stress.py - [line:336] - root - INFO - Successfully got vf device name of vm1: ens5
2022-02-11 09:53:52 - sshInstance.py - [line:92] - root - INFO - ifconfig ens5 192.168.10.13
root@localhost:~# 
2022-02-11 09:53:55 - test_network_sriov_stress.py - [line:283] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-02-11 09:54:01 - sshInstance.py - [line:92] - root - INFO - ifconfig ens5 down
root@localhost:~# 
2022-02-11 09:54:11 - sshInstance.py - [line:92] - root - INFO - ifconfig ens5 hw ether 1234567890ab
root@localhost:~# 
2022-02-11 09:54:20 - sshInstance.py - [line:92] - root - INFO - ifconfig ens5 up
root@localhost:~# 
2022-02-11 09:54:29 - sshInstance.py - [line:92] - root - INFO - nmcli device connect ens5
Command 'nmcli' not found, but can be installed with:
apt install network-manager
root@localhost:~# 
2022-02-11 09:54:38 - sshInstance.py - [line:92] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.13  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::1034:56ff:fe78:90ab  prefixlen 64  scopeid 0x20<link>
        ether 12:34:56:78:90:ab  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 9  bytes 766 (766.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
root@localhost:~# 
2022-02-11 09:54:47 - sshInstance.py - [line:92] - root - INFO - exit
logout
Connection to 192.168.122.130 closed.
[root@WW25ICX2 ~]# 
2022-02-11 09:54:54 - sshInstance.py - [line:69] - root - INFO - ping -c 5 192.168.122.130
PING 192.168.122.130 (192.168.122.130) 56(84) bytes of data.
64 bytes from 192.168.122.130: icmp_seq=1 ttl=64 time=0.207 ms
2022-02-11 09:54:55 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 192.168.122.130: icmp_seq=2 ttl=64 time=0.344 ms
2022-02-11 09:54:56 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 192.168.122.130: icmp_seq=3 ttl=64 time=0.257 ms
2022-02-11 09:54:57 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 192.168.122.130: icmp_seq=4 ttl=64 time=0.458 ms
2022-02-11 09:54:58 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 192.168.122.130: icmp_seq=5 ttl=64 time=0.307 ms

--- 192.168.122.130 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4121ms
rtt min/avg/max/mdev = 0.207/0.314/0.458/0.087 ms
[root@WW25ICX2 ~]# 
2022-02-11 09:55:07 - sshInstance.py - [line:92] - root - INFO - ssh root@192.168.122.130
root@192.168.122.130's password: 
2022-02-11 09:55:16 - sshInstance.py - [line:92] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Thu 10 Feb 2022 09:04:04 PM EST
  System load:  0.0                Processes:             131
  Usage of /:   38.9% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.130
  Swap usage:   0%                 IPv4 address for ens5: 192.168.10.13

372 updates can be installed immediately.
227 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Thu Feb 10 21:02:09 2022 from 192.168.122.1
root@localhost:~# 
2022-02-11 09:55:25 - sshInstance.py - [line:92] - root - INFO - nohup iperf -s -V&
[1] 1062
root@localhost:~# 
root@localhost:~# nohup: ignoring input and appending output to 'nohup.out'

2022-02-11 09:55:34 - sshInstance.py - [line:92] - root - INFO - exit
logout
Connection to 192.168.122.130 closed.
[root@WW25ICX2 ~]# 
2022-02-11 09:55:43 - sshInstance.py - [line:92] - root - INFO - nohup iperf -s -V&
[1] 824196
[root@localhost ~]# 
[root@localhost ~]# nohup: ignoring input and appending output to 'nohup.out'

2022-02-11 09:55:50 - sshInstance.py - [line:69] - root - INFO - ping -c 5 192.168.122.233
PING 192.168.122.233 (192.168.122.233) 56(84) bytes of data.
64 bytes from 192.168.122.233: icmp_seq=1 ttl=64 time=0.213 ms
2022-02-11 09:55:51 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 192.168.122.233: icmp_seq=2 ttl=64 time=0.328 ms
2022-02-11 09:55:52 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 192.168.122.233: icmp_seq=3 ttl=64 time=0.500 ms
2022-02-11 09:55:53 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 192.168.122.233: icmp_seq=4 ttl=64 time=0.470 ms
2022-02-11 09:55:54 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 192.168.122.233: icmp_seq=5 ttl=64 time=0.489 ms

--- 192.168.122.233 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4122ms
rtt min/avg/max/mdev = 0.213/0.400/0.500/0.112 ms
[root@WW25ICX2 ~]# 
2022-02-11 09:56:03 - sshInstance.py - [line:92] - root - INFO - ssh root@192.168.122.233
root@192.168.122.233's password: 
2022-02-11 09:56:12 - sshInstance.py - [line:92] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Thu 10 Feb 2022 09:05:00 PM EST
  System load:  0.0                Processes:             127
  Usage of /:   39.1% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.233
  Swap usage:   0%                 IPv4 address for ens5: 192.168.10.12

372 updates can be installed immediately.
227 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Thu Feb 10 21:00:12 2022 from 192.168.122.1
root@localhost:~# 
2022-02-11 09:56:21 - sshInstance.py - [line:92] - root - INFO - rm -rf /tmp/*.txt
root@localhost:~# 
2022-02-11 11:56:48 - sshInstance.py - [line:92] - root - INFO - iperf -c 192.168.10.13 -i 1 -f g -P 9 -t 7200 > /tmp/iperf_ipv4_vm1_vm2.txt
root@localhost:~# 
2022-02-11 13:57:14 - sshInstance.py - [line:92] - root - INFO - iperf -c 192.168.10.11 -i 1 -f g -P 9 -t 7200 > /tmp/iperf_ipv4_vm1_sut2.txt
root@localhost:~# 
2022-02-11 13:57:23 - sshInstance.py - [line:92] - root - INFO - exit
logout
Connection to 192.168.122.233 closed.
[root@WW25ICX2 ~]# 
2022-02-11 13:57:32 - sshInstance.py - [line:92] - root - INFO - rm -rf /tmp/*.txt
[root@WW25ICX2 ~]# 
2022-02-11 15:57:59 - sshInstance.py - [line:92] - root - INFO - iperf -c 192.168.10.11 -i 1 -f g -P 9 -t 7200 > /tmp/iperf_ipvv4_sut1_sut2.txt
[root@WW25ICX2 ~]# 
2022-02-11 15:58:08 - sshInstance.py - [line:92] - root - INFO - scp root@192.168.122.233:/tmp/*.txt /tmp
root@192.168.122.233's password: 
2022-02-11 15:58:17 - sshInstance.py - [line:92] - root - INFO - 
iperf_ipv4_vm1_sut2.txt                         0%    0     0.0KB/s   --:-- ETAiperf_ipv4_vm1_sut2.txt                       100% 3707KB 162.8MB/s   00:00    
iperf_ipv4_vm1_vm2.txt                          0%    0     0.0KB/s   --:-- ETAiperf_ipv4_vm1_vm2.txt                        100% 3707KB 158.0MB/s   00:00    
[root@WW25ICX2 ~]# 
2022-02-11 15:58:20 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: rm -rf /tmp/*.txt....
2022-02-11 15:58:20 - commonFunc.py - [line:30] - root - INFO - 
2022-02-11 15:58:20 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: rm -rf /tmp/*.txt
2022-02-11 15:58:20 - test_network_sriov_stress.py - [line:773] - root - INFO - Successfully iperf ipv4 vm1 vm2
2022-02-11 15:58:20 - test_network_sriov_stress.py - [line:773] - root - INFO - Successfully iperf ipv4 vm1 sut2
2022-02-11 15:58:20 - test_network_sriov_stress.py - [line:773] - root - INFO - Successfully iperf ipv4 sut1 sut2
2022-02-11 15:58:20 - test_network_sriov_stress.py - [line:774] - root - INFO - check ---end---
.2022-02-11 15:58:20 - test_network_sriov_stress.py - [line:57] - root - INFO - this is the teardown of Test_Network_SRIOV_Stress.
2022-02-11 15:58:23 - sshInstance.py - [line:44] - root - INFO - virsh destroy VMtest2
2022-02-11 15:58:23 - sshInstance.py - [line:48] - root - INFO - Domain VMtest2 destroyed


2022-02-11 15:58:28 - test_network_sriov_stress.py - [line:258] - root - INFO - Successfully shut down the VM: VMtest2
2022-02-11 15:58:31 - sshInstance.py - [line:44] - root - INFO - virsh undefine VMtest2
2022-02-11 15:58:31 - sshInstance.py - [line:48] - root - INFO - Domain VMtest2 has been undefined


2022-02-11 15:58:36 - test_network_sriov_stress.py - [line:268] - root - INFO - Successfully removed the VM: VMtest2
2022-02-11 15:58:39 - sshInstance.py - [line:44] - root - INFO - virsh destroy VMtest1
2022-02-11 15:58:40 - sshInstance.py - [line:48] - root - INFO - Domain VMtest1 destroyed


2022-02-11 15:58:45 - test_network_sriov_stress.py - [line:258] - root - INFO - Successfully shut down the VM: VMtest1
2022-02-11 15:58:48 - sshInstance.py - [line:44] - root - INFO - virsh undefine VMtest1
2022-02-11 15:58:48 - sshInstance.py - [line:48] - root - INFO - Domain VMtest1 has been undefined


2022-02-11 15:58:53 - test_network_sriov_stress.py - [line:268] - root - INFO - Successfully removed the VM: VMtest1
2022-02-11 15:58:53 - sshInstance.py - [line:39] - root - INFO - closing ssh client on SUT
2022-02-11 15:58:53 - test_network_sriov_stress.py - [line:63] - root - INFO - =============================================Finished run the test case===============================================


=============================== warnings summary ===============================
npxtester/Tests/testNics/test_iperf_client_4hour.py:37
  /root/.jenkins/workspace/test-test-iperf-SRIOV-ipv4-stress-2hours/npxtester/Tests/testNics/test_iperf_client_4hour.py:37: PytestUnknownMarkWarning: Unknown pytest.mark.iperf_tcp - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.iperf_tcp

npxtester/Tests/testNics/test_iperf_client_4hour.py:70
  /root/.jenkins/workspace/test-test-iperf-SRIOV-ipv4-stress-2hours/npxtester/Tests/testNics/test_iperf_client_4hour.py:70: PytestUnknownMarkWarning: Unknown pytest.mark.iperf_udp - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.iperf_udp

npxtester/Tests/testNics/test_mtu.py:40
  /root/.jenkins/workspace/test-test-iperf-SRIOV-ipv4-stress-2hours/npxtester/Tests/testNics/test_mtu.py:40: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_mtu.py:79
  /root/.jenkins/workspace/test-test-iperf-SRIOV-ipv4-stress-2hours/npxtester/Tests/testNics/test_mtu.py:79: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_network_sriov_stress.py:362
  /root/.jenkins/workspace/test-test-iperf-SRIOV-ipv4-stress-2hours/npxtester/Tests/testNics/test_network_sriov_stress.py:362: PytestUnknownMarkWarning: Unknown pytest.mark.nicSriov5times - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSriov5times

npxtester/Tests/testNics/test_network_sriov_stress.py:478
  /root/.jenkins/workspace/test-test-iperf-SRIOV-ipv4-stress-2hours/npxtester/Tests/testNics/test_network_sriov_stress.py:478: PytestUnknownMarkWarning: Unknown pytest.mark.nicSriov2hours - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSriov2hours

npxtester/Tests/testNics/test_network_sriov_stress.py:776
  /root/.jenkins/workspace/test-test-iperf-SRIOV-ipv4-stress-2hours/npxtester/Tests/testNics/test_network_sriov_stress.py:776: PytestUnknownMarkWarning: Unknown pytest.mark.SriovVfIperf - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.SriovVfIperf

npxtester/Tests/testNics/test_nics.py:39
  /root/.jenkins/workspace/test-test-iperf-SRIOV-ipv4-stress-2hours/npxtester/Tests/testNics/test_nics.py:39: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:64
  /root/.jenkins/workspace/test-test-iperf-SRIOV-ipv4-stress-2hours/npxtester/Tests/testNics/test_nics.py:64: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:87
  /root/.jenkins/workspace/test-test-iperf-SRIOV-ipv4-stress-2hours/npxtester/Tests/testNics/test_nics.py:87: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:115
  /root/.jenkins/workspace/test-test-iperf-SRIOV-ipv4-stress-2hours/npxtester/Tests/testNics/test_nics.py:115: PytestUnknownMarkWarning: Unknown pytest.mark.test - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.test

npxtester/Tests/testNics/test_nics.py:154
  /root/.jenkins/workspace/test-test-iperf-SRIOV-ipv4-stress-2hours/npxtester/Tests/testNics/test_nics.py:154: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

-- Docs: https://docs.pytest.org/en/stable/warnings.html
- generated html file: file:///root/testconfig/testlogs/HtmlReport/sriov_iperfvf_2hours_ipv4.html -
========= 1 passed, 15 deselected, 12 warnings in 22275.86s (6:11:15) ==========
[8mha:////4LYnTL2kHEK3xk4r0Ko1+Seo0jzleqkNHqW+48vbd8D5AAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiGoSEqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDyFBu95/avQHhgNoTvgAAAA==[0m[Pipeline] }
[8mha:////4NMYSizRLQ1dZuwWSEHzgQ6w9lPsBbHRCtMg5H1rEwslAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiCgqEqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDyFBu95/avQH+Ao9BvgAAAA==[0m[Pipeline] // stage
[8mha:////4DKXsabp5/afXFSsWLUBDHMQEXD3jgAzV8WEfOF26pEaAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMJBiA5R0VppeIFJjHFi3QX7QlLxIr7GH4iIRMVWO9PM641lijhydKqx1HpKlVdd6N301MCxvQYeVMMXVTElDlaVdii5tqcZSxaLeVmOhcbKUhU4eXKCtW7MwxTBkCvOEid30Mh9fccTmZ7KYqJ8YYzY3Po6Mf06fwMYu06Q77aCbP8BK1nRVL0AAAA=[0m[Pipeline] }
[8mha:////4HmorPDDfMRqsF6zz9/2vCOvXnWMSW4cx9qE6GTxL9MFAAAAoh+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIRUCIqWisNLzCJMU6su2BfSCpexNf4AxGRqNhqZ5p5vbFMEUeOTjWWWk+p8qoLvZueGji218CDaviiKqbEwarSDiXX9jRjyWIxL8ux0FhZqgInT06w1o15mCIYcsVZ4uQOGrmv73gi01NZTJQvjBGbW18npl/nbwBj1wny3VaQ7T845dX3vQAAAA==[0m[Pipeline] // node
[8mha:////4BE+fprF9eommgtvg+AAT8POwnI+05QgEWtQ8SEssetUAAAAoh+LCAAAAAAAAP9tjbEOgjAURS8YB1dHP6IkrMbJtWHxCyrUWmjeg/YhTH6Rv+Y/SCRx8k73nOW83timiBNHp1pLnadUe9WH0S1PTRy7W+BJtXxVNVPiYFVlp4obe16xYrFYl+XYaOws1YGTJyfY69Y8TBEMueIicXFHjdw3A57I9FIWE+ULc8ThPjaJ6df5G8Dc94K8LAVZ+QF+r3LHvQAAAA==[0m[Pipeline] End of Pipeline
Finished: SUCCESS
